hibernate-reactive-core/src/main/java/org/hibernate/reactive/engine/impl/ReactiveEntityUpdateAction.java

```diff
@@ -156,8 +155,7 @@ private void handleDeleted(EntityEntry entry, EntityPersister persister, Object
 		}
 	}
 
-	// TODO: copy/paste from superclass (make it protected)
-	private void handleNaturalIdResolutions(EntityPersister persister, SharedSessionContractImplementor session, Object id) {
+	protected void handleNaturalIdResolutions(EntityPersister persister, SharedSessionContractImplementor session, Object id) {
```

Comment:
All the TODO that you have removed are about making the method in ORM protected, so that we can remove them from Hibernate Reactive